Sources/swift-algorand-sdk/v2/client/IndexerClient.swift

```diff
@@ -54,6 +54,10 @@ public class IndexerClient {
         return LookUpAssetById(client: self, id: id)
     }
     
+    public func lookUpApplicationLogsById(id:Int64)-> LookUpApplicationLogsById{
+        return LookUpApplicationLogsById(client: self, applicationId: id)
+    }
+    
```

Comment:
Add **all** the parameters needed by `LookUpApplicationLogsById` in the public method. Use default values if they are not required. Avoid the "builder syntax" if possible.